src/SWP/Bundle/AnalyticsBundle/DependencyInjection/Configuration.php

```diff
@@ -30,10 +36,44 @@ class Configuration implements ConfigurationInterface
     public function getConfigTreeBuilder()
     {
         $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('swp_analytics', 'array');
-
-        $rootNode
+        $treeBuilder->root('swp_analytics')
             ->children()
+                ->arrayNode('persistence')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('orm')
+                            ->addDefaultsIfNotSet()
+                            ->canBeEnabled()
+                            ->children()
+                                ->arrayNode('classes')
+                                ->addDefaultsIfNotSet()
+                                ->children()
+                                    ->arrayNode('article_statistics')
+                                        ->addDefaultsIfNotSet()
+                                        ->children()
+                                            ->scalarNode('model')->cannotBeEmpty()->defaultValue(ArticleStatistics::class)->end()
```

Comment:
Names of models classes should not be plural according to our existing class naming.